### PR TITLE
chore: local multi node test setting files

### PR DESCRIPTION
**Pull request by leesj9476**
## Description

update init_node.sh and README.md file for local multi node test

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by egonspace — APPROVED** [review]
I confirmed that this script performs well.

File: init_node.sh [hunk 1]
```diff
@@ -1,88 +1,155 @@
...
+      sed -i 's/allow_duplicate_ip = false/allow_duplicate_ip = true/g' ${CHAIN_DIR}/config/config.toml
+
+      sed -i 's#'"${MEMO}"'#'"${MEMO_SPLIT[0]}"':'"${P2P_PORT}"'#g' ${CHAIN_0_DIR}/config/config.toml
+
+      sed -i 's/pruning = "default"/pruning = "nothing"/g' ${CHAIN_DIR}/config/app.toml
+      sed -i 's#"0.0.0.0:9090"#"0.0.0.0:'"${GRPC_PORT}"'"#g' ${CHAIN_DIR}/config/app.toml
+    else
+      sed -i '' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:'"${RPC_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
+      sed -i '' 's#"tcp://0.0.0.0:26656"#"tcp://0.0.0.0:'"${P2P_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
+      sed -i '' 's#"localhost:6060"#"localhost:'"${P2P_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
```
Comment (egonspace) [inline init_node.sh:135]:
```suggestion
      sed -i '' 's#"localhost:6060"#"localhost:'"${PROF_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
```

File: README.md [hunk 2]
```diff
@@ -36,17 +36,26 @@ To learn about Cosmos SDK, please refer [Cosmos SDK Docs](https://github.com/cos
...
-simd version    # you can see the version!
+
+# you can see the version!
+simd version
 ```
 
 **Configure**
 ```
-sh init_node.sh sim
+sh init_node.sh sim {N(number of nodes), default=1}
```
Comment (0Tech) [inline README.md:47]: I think you should replace "sh" into "zsh". Some lines of the script contain zsh specific features which cause errors on dash.

File: init_node.sh [hunk 3]
```diff
@@ -1,88 +1,155 @@
...
+    else
+      cp ${SRC_GENESIS_FIlE} "${CHAIN_DIR}/config"
+    fi
+
+    MONIKER="${MONIKER_PREFIX}${i}"
+    MEMO=`sed 's/"//g' <<< \`cat ${GENTXS_DIR}/${MONIKER}.json | jq '.body.memo'\``
+    MEMO_SPLIT=(`echo ${MEMO} | tr ":" "\n"`)
+
+    # Set proper defaults and change ports (use a different sed for Mac or Linux)
+    if [ "`uname`" == 'linux' ]; then
```
Comment (0Tech) [inline init_node.sh:117]: Please change 'linux' into 'Linux'.

File: init_node.sh [hunk 4]
```diff
@@ -1,88 +1,155 @@
...
+      sed -i 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:'"${RPC_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
+      sed -i 's#"tcp://0.0.0.0:26656"#"tcp://0.0.0.0:'"${P2P_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
+      sed -i 's#"localhost:6060"#"localhost:'"${P2P_PORT}"'"#g' ${CHAIN_DIR}/config/config.toml
+      sed -i 's/timeout_commit = "5s"/timeout_commit = "1s"/g' ${CHAIN_DIR}/config/config.toml
+      sed -i 's/timeout_propose = "3s"/timeout_propose = "1s"/g' ${CHAIN_DIR}/config/config.toml
+      sed -i 's/index_all_keys = false/index_all_keys = true/g' ${CHAIN_DIR}/config/config.toml
+      sed -i 's/persistent_peers = ""/persistent_peers = "${PERSISTANT_PEERS}"/g' ${CHAIN_DIR}/config/config.toml
```
Comment (0Tech) [inline init_node.sh:124]: Please use the same quotation trick as macOS's (PERSISTANT_PEERS variable is not evaluated).

**Review by 0Tech — CHANGES_REQUESTED** [review]
Applying the comments in my environment (Debian), it looks like working. :+1:

File: init_node.sh (hunk 3 above)
Comment (leesj9476) [reply init_node.sh:117]: thx 👍

File: init_node.sh (hunk 4 above)
Comment (leesj9476) [reply init_node.sh:124]: persistent_peer line is useless, I'll remove it

**Review by 0Tech — APPROVED** [review]
I checked it working :+1:

**Pull request merged**